clients/gitlabrepo/issues.go

```diff
@@ -48,8 +48,8 @@ func (handler *issuesHandler) setup() error {
 
 		// There doesn't seem to be a good way to get user access_levels in gitlab so the following way may seem incredibly
 		// barberic, however I couldn't find a better way in the docs.
-		projectAccessTokens, resp, err := handler.glClient.ProjectAccessTokens.ListProjectAccessTokens(
-			handler.repourl.project, &gitlab.ListProjectAccessTokensOptions{})
+		projMemberships, resp, err := handler.glClient.ProjectMembers.ListAllProjectMembers(
```

Comment:
Does this work with a public scoped token?

Reply:
Yes, it does.